### ISSUE TITLE
Contao 5.X compatibility

### ISSUE THREAD

**Issue by githuppi** [markocupic/contao-multifile-download#3]
Hello,

is it possible to make your extension compatible with Contao 5.X? Currently it can't be installed.

Thanks for a short response.

**Comment by markocupic** [comment]
@githuppi 
Yes, of course. If you like, you are welcome to do the job yourself and then submit your work with a pull request. Otherwise you can get in touch with me via [email ](https://github.com/markocupic/contao-multifile-download/blob/d220f3953c5e847644d8fb42e23d7518b3b84e90/src/ContaoManager/Plugin.php#L8)and I will do the update for you as part of a small order.

**Comment by githuppi** [comment]
Thanks for the quick response. 

Since I am not familiar with changing the code to submit a pull request I had to switch to another extension which provided the same functionality.

Kind regards and happy coding.

**Comment by markocupic** [comment]
Version 2.x should run under Contao 5.x
